spec/controllers/videos_controller_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+
+describe VideosController do
+  describe "video manipulation" do
+    login_admin
+
+    before(:each) do
+      @video = FactoryGirl.create(:video)
+    end
+
+    it "can allow an admin to update a video" do
+      put 'update', :id => @video.id, :video => { 'title' => 'New Title' } 
+      expect(@video.reload.title).to eq 'New Title'
+    end
+
+    it "cannot allow an admin to erase all attributes" do
```

Comment:
Bear in mind that this doesn't actually erase any attributes. Passing in empty params to `update` does nothing.